src/components/Captcha.vue

```diff
@@ -1,137 +1,324 @@
+                  begin="0s"
+                  repeatCount="indefinite"
+                ></animate>
               </rect>
             </g>
           </svg>
           <span>Play audio</span>
         </button>
-        <button :data-cy="getCypressValue('TryAnother')" class="captcha-button try-another-image" href="javascript:void(0)" @click="handleTryAnotherImageClick()" role="button">
-          <svg class="icon-loop"><use xlink:href="#icon-loop-def"></use></svg>
+        <button
+          :data-cy="getCypressValue('TryAnother')"
+          class="captcha-button try-another-image"
+          href="javascript:void(0)"
+          @click="handleTryAnotherImageClick()"
+          role="button"
+        >
+          <svg class="icon-loop">
+            <use xlink:href="#icon-loop-def"></use>
+          </svg>
           <span>Try another image</span>
         </button>
       </div>
     </div>
     <div class="captcha-input-container">
       <div>
-        <label for="input-answer">Enter the text you either see in the box or you hear in the audio</label>
+        <label for="input-answer"
+          >Enter the text you either see in the box or you hear in the
+          audio</label
+        >
       </div>
-      <input id="input-answer"
-            :class="'form-control input-answer ' + (isInputValid === false ? 'border-danger' : '')"
-            v-model="inputAnswer"
-            :data-cy="getCypressValue('Input')"
-            :disabled="(inputAnswer && inputAnswer.length === 6 && isLoadingCaptchaVerification) ? true : false"
-            @input="handleInputChange($event)"
-            @blur="handleBlur($event)"
-            maxlength="6"
-            autocorrect="off"
-            autocomplete="off"
-            autocapitalize="none"
-            aria-required="true" />
-      <div v-if="isLoadingCaptchaVerification"
-          class="validation-spinner-container">
-        <Loader :color="'#AAA'"
-                :size="'20px'" />
+      <input
+        id="input-answer"
+        :class="
+          'form-control input-answer ' +
+          (isInputValid === false ? 'border-danger' : '')
+        "
+        v-model="inputAnswer"
+        :data-cy="getCypressValue('Input')"
+        :disabled="
+          inputAnswer &&
+          inputAnswer.length === 6 &&
+          isLoadingCaptchaVerification
+            ? true
+            : false
+        "
+        @input="handleInputChange($event)"
+        @blur="handleBlur($event)"
+        maxlength="6"
+        autocorrect="off"
+        autocomplete="off"
+        autocapitalize="none"
+        aria-required="true"
+      />
+      <div
+        v-if="isLoadingCaptchaVerification"
+        class="validation-spinner-container"
+      >
+        <Loader :color="'#AAA'" :size="'20px'" />
       </div>
     </div>
-    <div v-if="errorMessage"
-        class="error-message mt-2 text-danger">{{errorMessage}}</div>
+    <div v-if="errorMessage" class="error-message mt-2 text-danger">
```

Comment:
Is it possible to have each attribute on its own line regardless of line length?